docker/Dockerfile.faux

```diff
@@ -45,6 +46,28 @@ COPY subset/bacnet/bacnetTests/src/main/resources/Faux*.json tmp/
 COPY --from=java_build /root/bacnet4j/bacnet4j-1.0-SNAPSHOT-all.jar bacnetTests/libs/
 RUN cd bacnetTests && ./gradlew build
 
+#Telnet dependancy
+RUN $AG update && $AG install -y telnetd
+RUN $AG update && $AG install xinetd
+RUN service xinetd start
+
+#SSH dependancy
+COPY subset/security/ssh.config ssh.config
```

Comment:
why not copy to /etc/ssh directly?

Reply:
This file does not replace the ssh.config in the container, it merely amends to it, which is why I copy that file over and then use the below RUN command to attach the contents of that file to the bottom of the faux devices ssh config. Will change the file name so that it doesn't instead so that it's clear the file isn't a replacement.